client/src/views/SearchResults.tsx

```diff
@@ -104,7 +104,7 @@ function SearchResults(): JSX.Element {
     } else if (querySearchCategory === 'Offers' || querySearchCategory === 'Needs') {
       fetch(
         `${APP_API_BASE_URL}/assets?type=${
-          querySearchCategory === 'Needs' ? 'donation' : 'request'
+          querySearchCategory === 'Needs' ? 'request' : 'donation'
```

Comment:
The preferred url and query param creation is with [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams).

Reply:
😮‍💨🌴 i was wondering why the search bar wasn't returning the results i expected.....
great catch